wicket-jquery-ui/src/main/java/com/googlecode/wicket/jquery/ui/widget/tabs/TabsBehavior.java

```diff
@@ -361,7 +368,7 @@ public int getIndex()
 	/**
 	 * Provides an event object that will be broadcasted by the {@link OnActivatingAjaxBehavior} callback
 	 */
-	protected static class ActivatingEvent extends TabEvent
+	protected static class ActivatingEvent extends ActivateEvent
```

Comment:
Is there a particular reason for extending `ActivateEvent` instead of `TabEvent`?

Reply:
Ok, I got it! :)

Reply:
This is because of this code: https://github.com/sebfz1/wicket-jquery-ui/blob/aebcb069cccbad6330dcbcd6809194fcbfebabd6/wicket-jquery-ui/src/main/java/com/googlecode/wicket/jquery/ui/widget/tabs/TabsBehavior.java#L214

Sorry was unable to answer earlier :)